cabal.project

```diff
@@ -20,10 +20,9 @@ repository cardano-haskell-packages
 -- rather than a shortened one.
 source-repository-package
   type: git
-  location: https://github.com/IntersectMBO/formal-ledger-specifications.git
-  subdir: ledgerSrc/haskell/Ledger
-  tag: 841942c68993857fd40ca35cd62258cf82d160a5
-  --sha256: 0acbsfpa3f745abfagyivim3d2qp7i0xwv0lsg7j4ah4s2qx5vkc
+  location: https://github.com/input-output-hk/cardano-ledger-executable-spec.git
```

Comment:
I think we might want to get rid of this repo, so the new way that @Lucsanszky has been working on, is to use the formal-ledger-specifications.git repo. Unless maybe you want to use something that hasn't been merged?

Reply:
I'll have to wait for the branch to get merged into `master` in the spec repo for that